src/main/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/CreateTaskRequest.java

```diff
@@ -22,6 +31,10 @@ public Transition getTransition() {
         return transition;
     }
 
+    public String getDueDate() {
```

Comment:
suggestion: Lombok Getter annotation.

Reply:
We decided not to use Lombok the other day I did say at the time that it would avoid the code coming up in code coverage but after talking about it the decision was not to use Lombok.